### PR TITLE
Add descriptions for ROOT GOF and unfolding projects

### PR DESCRIPTION
**Pull request by lmoneta**

### CODE REVIEW

**Comment by hegner** [comment]
Could you as well add some description for the university of iceland? Something like done in https://raw.githubusercontent.com/HEP-SF/hep-sf.github.io/master/_gsocorgs/2018/princeton.md ?

**Comment by lmoneta** [comment]
Here is the description: 

The University of Iceland is an international research university and a leading research istitution in Iceland.

Attached is the logo:
universityiceland_logo

File: _gsocprojects/2018/proposal_ROOTunfolding.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Implementation of a new Unfolding Procedure
+layout: gsoc_proposal
+project: ROOT
+year: 2018
+organization: 
+  - CERN
+  - University of Iceland
```
Comment (jouvin) [inline _gsocprojects/2018/proposal_ROOTunfolding.md]: Organization name must be a single word (no whitespace).

**Comment by jouvin** [comment]
@lmoneta BTW, if you rebase your PR on upstream/master, you should get rid of the error related to GenToo.

**Pull request merged**